static/js/components/LearnerSearch.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {
+  SearchkitComponent,
+  Hits,
+  NoHits,
+  MenuFilter,
+  SelectedFilters,
+  RefinementListFilter,
+  HitsStats,
+  Pagination,
+} from 'searchkit';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import Card from 'react-mdl/lib/Card/Card';
+import Dropdown from 'react-dropdown';
+
+import LearnerResult from './search/LearnerResult';
+import CountryRefinementOption from './search/CountryRefinementOption';
+import FilterVisibilityToggle from './search/FilterVisibilityToggle';
+import HitsCount from './search/HitsCount';
+import type { Option } from '../flow/generalTypes';
+
+export default class LearnerSearch extends SearchkitComponent {
+  props: {
+    checkFilterVisibility:  (s: string) => boolean,
+    setFilterVisibility:    (s: string, v: boolean) => void,
+  };
+
+  dropdownOptions: Option[] = [
+    { value: 'name', label: "Sort: name" },
+    { value: 'dob', label: "Sort: dob" },
+  ];
+
+  profileFieldOptions: Function = (): Object => ({
+    type: 'nested',
+    options: {
+      path: 'profile'
+    }
+  });
```

Comment:
It probably won't go away, based on the conversation Giovanni and I had this morning. Searchkit basically assumes in it's design that it's communicated directly with something that 'speaks' elasticsearch, so our backend is probably going to more or less pass queries straight through to ES and return the results back. All we'll really be doing on the search API is doing some examination of the query in question to see if the user has permissions to perform it - if they do we pass it along to ES and then return the result, if they do not we throw up a 401 or something.

does that sound accurate @giocalitri ?

I wrote this as a function because once the options object is passed in to the searchkit components they mutate it. this could be a static field and I could use something like `_.cloneDeep` on it instead.